.rubocop.yml

```diff
@@ -763,22 +757,21 @@ Metrics/PerceivedComplexity:
 ### Warnings
 
 Lint/AmbiguousOperator:
-  Description: >-
-                 Checks for ambiguous operators in the first argument of a
-                 method invocation without parentheses.
   Enabled: false
 
+# TODO: 2018/08/20 Discuss at next style guide meeting
+# Pretty sure we want to enable?
```

Comment:
This is what I'm doing to remind myself to raise at the SepMoCoReStyGuCo.
Thoughts?